lib/fpm/command.rb

```diff
@@ -117,6 +117,9 @@ def help(*args)
     "configuration files, specify this flag multiple times.  If argument is " \
     "directory all files inside it will be recursively marked as config files.",
     :multivalued => true, :attribute_name => :config_files
+  option "--no-config-files", :flag,
```

Comment:
At first, I thought that it would be the same for RPM, but all spec files seem to have to declare the files as a configuration file (with `%config`). So, I will change the option to be Debian specific.